src/runtime/runtime_cortexm_hardfault_config.go

```diff
@@ -0,0 +1,27 @@
+// +build nxp,!mimxrt1062
```

Comment:
Seems to me the contents of this file should be added to a new file in the  `device` package similar to what you did for https://github.com/tinygo-org/tinygo/tree/release/src/device/stm32

Reply:
Moved to new file `src/device/nxp/hardfault.go`